src/System.Numerics.Vectors/tests/GenericVectorTests.tt

```diff
@@ -1867,6 +1867,16 @@ namespace System.Numerics.Tests
             }
             throw new NotSupportedException();
         }
+
+        internal static T[] GetArrayOfDefaultValues<T>(int count)
+        {
+            T[] arr = new T[count];
+            for (int index = 0; index < count; ++index)
+            {
+                arr[index] = default(T);
```

Comment:
C# sets all the elements of a new array to the default value implicitly.  We shouldn't need to reset them here.  Instead we should just be able to call `new T[count]` in the calling code and get rid of this method completely.

Reply:
Done.